### ISSUE TITLE
Parameter Error

### ISSUE THREAD

**Issue by Cixap** [bendodson/Apple-TV-Alarm-Clock#3]
Hi! 

First of all, thank you so much for this script, it does exactly what I was looking for. 
With this said, I'm having this issue: 

If I execute the script with iTunes already open, it will work perfectly, but if it is closed Script Editor will pop up this error: **iTunes got an Error: Parameter error** on line 50 (play playlist Playlistname). This is literally my first interaction with any kind of script ever so please forgive my total noobness.

Thank you again!

**Comment by bendodson** [comment]
Hmm, that shouldn't happen. The line `activate application "iTunes"` should open iTunes for you so it would always be running. What version of macOS / iTunes are you using? I've tested it with macOS Sierra 10.12.3 and iTunes 12.5.5.5 which are the latest public versions and it works with iTunes closed or running.

**Comment by Cixap** [comment]
Hi again, 

I finally figured out what it was, it was just that since I was trying to play a list with Apple Music songs iTunes wasn't given time enough to load the playlist and for the playlist to be playable (it works perfectly fine with downloaded songs), that's why the error was at `play playlist Playlistname`. I just inserted a wide `delay 20` to give it enough time to load the songs and now it works!

Thank you and sorry for your trouble.
